### DISCUSSION TITLE
Greenfield redering in sea looks like land

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#13760] [Maps Related]
A harbour is planning on doing an expansion some time in the future. Someone has mapped the planned construction area in the sea as landuse=greenfield (Undeveloped land scheduled to turn into a construction site). I noticed it in the app since in the OsmAnd-mapstyle and a few others the rendering of the area looks like regular land area. 

To avoid confusion would it perhaps make more sense to not render 'greenfield' on any of the maps?

The area in question https://www.openstreetmap.org/#map=14/56.1426/10.2453

**Comment by vshcherb** [comment]
https://osmand.net/map#16/56.1390/10.2133 you can browse Default HD, not sure if I understand what's the issue

**Comment by ERYpTION** [comment]
Using the app I see a large area on the map south east of the harbour (see attached screenshot) that to my eyes looks like land. It is in fact open water that the city is planning to fill for an expansion of the harbour sometime in the future. I am just wondering if it would be better not to render 'greenfield' like that. To someone not familiar with the area I imagine it could cause some confusion. Or am I misunderstanding something? 
![Screensho_osmand2](https://user-images.githubusercontent.com/80313840/154284703-21078d8a-b7c8-4317-b4b5-276c86192ecb.jpg)

**Reply by vshcherb** [reply]
On latest maps it looks different
![Screen Shot 2022-02-17 at 19 33 42](https://user-images.githubusercontent.com/1042025/154547810-38167cad-9d49-4026-af7d-f543964fdf77.png)

**Reply by vshcherb** [reply]
Or probably it's being fixed in latest version

**Reply by ERYpTION** [reply]
Having now updated to the latest version (4.2.4) I still see the greenfield rendering in the sea as shown in the screenshot. I dont know why the rendering is still not the same on the osmand.net map. I do realize that it is not a bug as such since the area is in fact tagged as landuse=greenfield. I just still don't think it is a good way to render greenfield in the sea in terms of usability.
